kornia/contrib/extract_patches.py

```diff
@@ -8,6 +8,55 @@
 PadType = Union[Tuple[int, int], Tuple[int, int, int, int]]
 
 
+def compute_padding(
+    original_size: Union[int, Tuple[int, int]], window_size: Union[int, Tuple[int, int]]
+) -> Tuple[int, int, int, int]:
+    r"""Compute required padding to ensure chaining of :class:`~kornia.contrib.ExtractTensorPatches` and
+    :class:`~kornia.contrib.CombineTensorPatches` produces expected result.
```

Comment:
Wasn't sure if I should refer to the class `ExtractTensorPatches` or the function `extract_tensor_patches`

Reply:
yeah, I'd prioritize functional api